89.md

```diff
@@ -0,0 +1,81 @@
+NIP-89
+======
+
+Bitcoin Transaction Package Relay
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains encoded bitcoin transactions.
+The bitcoin transactions are encoded as a base64 string, and are contained in a `transactions` tag.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transactions are for.
+The common ones are:
+
+- mainnet: f9beb4d9
+- testnet3: 0b110907
+- signet (default): 0a03cf40
+- regtest: fabfb5da
+
+If another value is present it is likely a custom signet.
+
+The `tags` field MUST contain a `transactions` tag that contains a list of bitcoin transactions encoded in base64.
```

Comment:
Why base64? Isn't bitcoin core's standard for transactions hex?

Reply:
Hex is so much more pleasant!

Reply:
Yeah did to save on bandwidth, since these are ephemeral I guess it is not a big deal since they shouldn't be stored.